src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java

```diff
@@ -89,14 +92,19 @@ public Object deserialize( JsonParser p, DeserializationContext ctxt ) throws IO
         StreamSupport.stream( beanDescription.getClassInfo().memberMethods().spliterator(), true ).forEach( m -> {
             if ( m.hasAnnotation( JsonPostDeserialize.class ) )
             {
-                logger.debug( "Invoking method {} ", m.getName() );
+                logger.info( "Invoking method {} on {} ", m.getName(), beanDescription.getBeanClass() );
```

Comment:
I would probably leave this as debug.

Reply:
Not sure .... using as a java library I would agree.When its being run inside jenkins I thinks it useful.

Reply:
I wonder if we supply a default `logging.properties` configured to debug (I think Jenkins uses JUL) and it works just for jenkins ?

Reply:
Reverted to debug ; will deal with Jenkins separately.